portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -103,13 +104,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
       // Get type of comment.
       String commentType = getParameter(request, "tags", "Default");
+      
+      //Get number of comments.
+      numberOfCommentsToDisplay = getNumberOfCommentsToDisplay(request);
 
       // Get the URL of the image that the user uploaded to Blobstore.
       String imageUrl = getUploadedFileUrl(request, "image");
 
       //Get replies.
-      ArrayList<String> messageReplies = getParameter(request, "replies", "").split(",");
-
+      String messageRepliesString = getParameter(request, "replies", "");
+	    String messageRepliesArray[] = messageRepliesString.split(",");
```

Comment:
I may be wrong but I think using split assumes that all replies are obtained in one round, and you're trying to separate them? Since some replies can come later, I think using params to get and add them to the ArrayList probably will be better

Reply:
I'm not sure if I understand, but the once the form is submitted all the data becomes static, so if we were to have a dynamic list of responses that updated regularly, we would need to implement some other functionality for that

Reply:
Oh okay, makes sense